src/lib/live/showcaseStore.svelte.ts

```
@@ -0,0 +1,702 @@
+		if (winSide === 'White') winner = 'w';
+		else if (winSide === 'Black') winner = 'b';
+		else if (
+			this.live.outcome === 'draw' ||
+			this.live.termination === 'Draw' ||
+			(this.lastOver && 'Draw' in this.lastOver.result)
+		) {
+			winner = 'draw';
+		}
+
+		const term = this.live.termination ?? this.lastOver?.termination ?? 'KingCaptured';
+		let reason: string = 'mate';
```

ESLint

[error] The value assigned to 'reason' is not used in subsequent statements.

(no-useless-assignment)